aseba/launcher/src/qml/Applications.qml

```diff
@@ -89,6 +89,8 @@ ListModel {
             Utils.launchOsXBundle("Thonny", {"uuid" : device.id,
                                                        "endpoint" : device.tcpEndpoint(),
                                                        "password" : device.password()})
+		} else if (Utils.platformIsLinux()) {
+            return Utils.launch_process("flatpak", ["run", "org.thonny.Thonny"])
```

Comment:
is that still needet?

Reply:
No. launch_thonny is never called on Linux and wouldn't work.

Reply:
so please remove